### PR TITLE
Right align and limit the announcements

### PR DESCRIPTION
**Pull request by realSquidCoder**
Announcements are now right aligned and limited to showing the 10 most recent ones. Will be a working user config in `init.txt` in future.

### CODE REVIEW

File: GUI.cpp [hunk 1]
```diff
@@ -333,7 +333,8 @@ void draw_report_border(const ALLEGRO_FONT *font, float x, float y, int flags, c
 
 void draw_announcements(const ALLEGRO_FONT *font, float x, float y, int flags, std::vector<df::report *> &announcements)
 {
-    for (int i = announcements.size() - 1; i >= 0 && announcements[i]->duration > 0; i--)
+    int maxAnnouncements = 10;
+    for (int i = announcements.size() - 1; i >= (announcements.size() - maxAnnouncements) && announcements[i]->duration > 0; i--)
```
Comment (myk002) [inline GUI.cpp:337]: will crash if there are fewer than 10 announcements in the list.

how about `int maxAnnouncements = std::min(10, (int)announcements.size());`?

Comment (realSquidCoder) [reply GUI.cpp:337]: thx i forgot that.

**Comment by myk002** [comment]
reminder to submit a separate PR for the changelog: https://github.com/DFHack/dfhack/blob/develop/docs/changelog.txt

File: .vscode/settings.json
Comment (myk002) [inline .vscode/settings.json:1]: I don't think this file should have been added to this PR

Comment (realSquidCoder) [reply .vscode/settings.json:1]: ugh how did that even get there. i might need to sleep on this, come back with fresh mind, im getting careless

**Review by myk002 — APPROVED** [review]

**Pull request merged**